src/sizzle.js

```diff
@@ -150,6 +150,24 @@ var i,
 				String.fromCharCode( high >> 10 | 0xD800, high & 0x3FF | 0xDC00 );
 	},
 
+	// CSS string/identifier serialization
+	// https://drafts.csswg.org/cssom/#common-serializing-idioms
+	rcssescape = /([\0-\x1f\x7f]|^-?\d)|^-$|[^\x80-\uFFFF\w-]/g,
+	fcssescape = function( ch, numeric ) {
+
+		// We allow NULL input, unlike the draft spec as of 2015-11-09
+		// Our syntax is already broader elsewhere
```

Comment:
I definitely think it's worth bringing up to the standards guys. Edge cases like this drive me crazy, I'd rather just have the browser close and show a dialog with "GAME OVER". :smiling_imp: